composer.json

```diff
@@ -5,6 +5,7 @@
   "license": ["MIT"],
   "version": "3.13.2",
   "require": {
+    "php": "~8.1|~8.2",
```

Comment:
Even though we don't officially support PHP 7, let's wait till 3.14 before we explicitly call out PHP 8.